packages/core/src/parser.ts

```diff
@@ -0,0 +1,155 @@
+// Copyright IBM Corp. 2017. All Rights Reserved.
+// Node module: @loopback/core
+// This file is licensed under the MIT License.
+// License text available at https://opensource.org/licenses/MIT
+
+import {ServerRequest as Request, ServerResponse as Response} from 'http';
+import {Application} from './Application';
+import {invoke} from './invoke';
+import {SwaggerRouter} from './router/SwaggerRouter';
+import {Endpoint} from './router/SwaggerRouter';
+import {OpenApiSpec, OperationObject, ParameterObject} from '@loopback/openapi-spec';
+import * as assert from 'assert';
+const debug = require('debug')('loopback:request-parser');
+import * as Promise from 'bluebird';
+import * as url from 'url';
+import * as pathToRegexp from 'path-to-regexp';
```

Comment:
Some of these imports are unused, e.g. `pathToRegexp`. It would be great to clean them up.